components/add_groups_to_team_modal/add_groups_to_team_modal.test.jsx

```diff
@@ -72,6 +74,7 @@ describe('components/AddGroupsToTeamModal', () => {
         wrapper.setState({values: []});
         await wrapper.instance().handleSubmit({preventDefault: jest.fn()});
         expect(actions.linkGroupSyncable).not.toBeCalled();
+        expect(instance.handleHide).not.toBeCalled();
```

Comment:
Lets add this here for completeness
```suggestion
        expect(instance.handleResponse).not.toBeCalled();
        expect(instance.handleHide).not.toBeCalled();
```